backend/.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 # Keep environment variables out of version control
 .env
+./tests
```

Comment:
i feel like it makes sense to include tests in the repo personally. also note if they arent here the rest of us cant access them!